test/e2e_node/runtime_conformance_test.go

```diff
@@ -41,7 +41,11 @@ var _ = SIGDescribe("Container Runtime Conformance Test", func() {
 	ginkgo.Describe("container runtime conformance blackbox test", func() {
 
 		ginkgo.Context("when running a container with a new image", func() {
-			// The service account only has pull permission
+			// For the future security scans:
+			// The service account only has pull permission.
+			// The image is only made private to use in this test.
```

Comment:
Clarification: repo is private, and images are all readily available public images, so nothing of value is protected by this credential.

// The container repo is only made private to test private container pulling
// All container images in the repo are public container images

Reply:
@SergeyKanzhelev WDYT about this?